mitodl/open-discussions
# Refactor profile upload to use withForm

#### What are the relevant tickets?
Part of #927 

#### What's this PR do?
Refactors the profile image component and container to user `withForm` and `redux-hammock`. This should be used next for the channel avatar and banner editor.

#### How should this be manually tested?
Edit the profile image, nothing should appear to work differently. View the profile image on the home page and in a post to verify that both editable and non-editable modes still work.
